src/lpython/parser/tokenizer.re

```diff
@@ -535,6 +536,7 @@ std::string token2text(const int token)
         T(TK_ATEQUAL, "@=")
         T(TK_RARROW, "->")
         T(TK_COLONEQUAL, ":=")
+        T(TK_DBL_COLON, ":=")
```

Comment:
```suggestion
        T(TK_DBL_COLON, "::")
```

Reply:
Good catch, Thank you for updating the same...